python/python/glide/async_commands/core.py

```diff
@@ -2306,6 +2267,72 @@ async def geopos(
             await self._execute_command(RequestType.GeoPos, [key] + members),
         )
 
+    async def geosearch(
+        self,
+        key: str,
+        search_from: Union[str, GeospatialData],
+        seach_by: Union[GeoSearchByRadius, GeoSearchByBox],
+        order_by: Optional[OrderBy] = None,
+        count: Optional[GeoSearchCount] = None,
+        with_coord: bool = False,
+        with_dist: bool = False,
+        with_hash: bool = False,
+    ) -> List[Union[str, List[Union[str, float, int, List[float]]]]]:
+        """
+        Searches for members in a sorted set stored at `key` representing geospatial data within a circular or rectengal area.
+
+        See https://valkey.io/commands/geosearch/ for more details.
```

Comment:
Add `Since` section.
Consider updating already implemented commands.

Reply:
>Consider updating already implemented commands.

Will do that in another pr with the rest of the changes like https://github.com/aws/glide-for-redis/pull/1482/files#r1617876469